charts/orchestrator/values-rc.yaml

```diff
@@ -0,0 +1,39 @@
+sonataFlowOperator:
+  isReleaseCandidate: true # Indicates RC builds should be used by the chart to install Sonataflow
+  enabled: true # whether the operator should be deployed by the chart
+  subscription:
+    namespace: openshift-serverless-logic # namespace where the operator should be deployed
+    channel: alpha # channel of an operator package to subscribe to
+    name: logic-operator-rhel8 # name of the operator package
+    sourceName: sonataflow-brew # name of the catalog source
+    startingCSV: logic-operator-rhel8.v1.33.0 # The initial version of the operator
+
+# rhdhOperator:
+#   isReleaseCandidate: false # Indicates RC builds should be used by the chart to install RHDH
+#   enabled: true # whether the operator should be deployed by the chart
+#   subscription:
+#     namespace: rhdh-operator # namespace where the operator should be deployed
+#     channel: fast # channel of an operator package to subscribe to
+#     name: rhdh # name of the operator package
+#     source: rhdh-brew # name of the catalog source
+#     startingCSV: rhdh-operator.v1.2.0 # The initial version of the operator
+
+sonataflow:
+  images:
+    4.13: brew.registry.redhat.io/rh-osbs/iib:747442
```

Comment:
should we support 4.12?
https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/administration_guide_for_red_hat_developer_hub/assembly-install-rhdh-ocp#proc-install-rhdh-airgapped-environment-ocp-helm_assembly-install-rhdh-ocp